src/cli/commands/append/appendManager.ts

```diff
@@ -222,7 +223,7 @@ export class AppendManager {
     });
 
     const simplifiedDiffPath = join(DATA_DIR, `${this.stateTracker.nextState}.simplified.${DIFF_FILE_EXTENTION}`);
-    await this.osmCommandRunner.mergeChanges([`${diffPath}`, `--output=${simplifiedDiffPath}`]);
+    await this.osmCommandRunner.mergeChanges([`${diffPath}`, `--output=${simplifiedDiffPath}`, `--overwrite`]);
```

Comment:
i may have forgot, but what does the --overwrite means in osm2pgsql , is it a new flag?

Reply:
it's an `osmium` flag that indicates it to overwrite output if already exists. this is relevant since now `osm2pgsql-wrapper` is deployed as `deployment` and not a `job`